.github/workflows/sign-and-release-to-github.yml

```diff
@@ -52,11 +52,36 @@ jobs:
         id: release_notes
         run: echo ::set-output name=release_notes::$(git log --no-merges --pretty=format:"%h %s" ${{ steps.version.outputs.version }}^..${{ steps.version.outputs.version }})
 
-      - name: "Release"
```

Comment:
Is this running the upload artifact as part of the release notes step?

Reply:
No, it's a new step (due to the `-`); it just doesn't have a name.